pkgs/top-level/aliases.nix

```diff
@@ -1412,6 +1412,7 @@ mapAliases ({
   ultrastardx-beta = throw "'ultrastardx-beta' has been renamed to/replaced by 'ultrastardx'"; # Converted to throw 2022-02-22
   unicorn-emu = unicorn; # Added 2020-10-29
   unifiStable = unifi6; # Added 2020-12-28
+  unity3d = throw "'unity3d' is unmaintained, has seen no updates in years and depends on deprecated GTK2"; # Added 2022-06-16
```

Comment:
```suggestion
  unity3d = throw "'unity3d' is unmaintained and has seen no updates in years, it also depends on GTK2 which is now deprecated"; # Added 2022-06-16
```

Reply:
This does not read like an improvement to me.

Reply:
Thanks, I'll leave it as-is.

Reply:
The way it is written, it looks like "a deprecated release of GTK2", rather than "GTK2, a deprecated library".

But I will not complain. Next months those aliases will be removed.